apps/passport-client/src/useSyncE2EEStorage.tsx

```diff
@@ -19,6 +21,40 @@ import {
 import { getPackages } from "./pcdPackages";
 import { useOnStateChange } from "./subscribe";
 
+export async function updateBlobKeyForEncryptedStorage(
+  oldEncryptionKey: string,
+  newEncryptionKey: string,
+  newSalt: string
+): Promise<ChangeBlobKeyResult> {
+  const user = loadSelf();
+  const pcds = await loadPCDs();
+  if (pcds.size() === 0) {
```

Comment:
That makes sense. This was initially put here to prevent a race condition bug where the user could have their initial semaphore identity PCD deleted (you can manually delete that PCD), but it didn't that this check had resolved that issue. Going to remove for both requests.

Reply:
Thanks for the history.  Agreed that fixing that bug would be better done in a more directed way by preventing deleting that particular PCD.  Preventing deleting the "last" PCD doesn't really prevent that.